extensions/git/src/commands.ts

```diff
@@ -1076,7 +1076,9 @@ export class CommandCenter {
 			.filter(r => r.type === Status.UNTRACKED || r.type === Status.IGNORED);
 		const uris = resources.map(r => r.resourceUri);
 
-		await repository.add(uris);
+		if (uris.length > 0) {
```

Comment:
Trying to fix the following scenario (when clicking the + when no changes):
![image](https://user-images.githubusercontent.com/1062699/81475888-186a3600-920f-11ea-9cb4-7896dee1ea8d.png)
The actual result: 
![image](https://user-images.githubusercontent.com/1062699/81475985-a34b3080-920f-11ea-9b60-a4816b6fdc37.png)
The expected result:
![image](https://user-images.githubusercontent.com/1062699/81476027-e0afbe00-920f-11ea-906a-6592d9356765.png)